src/vsphere_cpi/lib/cloud/vsphere/vm_creator.rb

```diff
@@ -114,6 +114,36 @@ def create(vm_config)
           raise "Unable to parse vmx options: 'vmx_options' is not a Hash"
         end
 
+        host = nil
+        # Before cloning we need to make sure the host is correctly picked up
+        # from cluster's host group if provided.
+        unless cluster.host_group.nil?
+          # placement_spec = VimSdk::Vim::Cluster::PlacementSpec.new
```

Comment:
since, this Since, this is for future implementation, should remove it from here before we merge to master

Reply:
I would prefer keeping it. It is small and is not making things messy. Will be super easy to uncomment and push later.

Plus, this PR might uncomment it later depending on test runs with and without this change today.